src/Api/PubnubApi/Enum/PushTypeService.cs

```diff
@@ -3,8 +3,9 @@ namespace PubnubApi
 {
     public enum PNPushType
     {
-        GCM,
```

Comment:
We need to make sure to document this if we're going to remove it. Will this break current users with this Type? Is it possible to leave it in as backwards compatibility?

Reply:
re-added GCM for backward-compatibility. So we will have both FCM and GCM